stats_calculator.py

```diff
@@ -19,14 +20,16 @@ def calculate_stat_from_level(base_stats: dict, stat_name: str, level: int):
     return calculate_flat_stat_from_level(stat, mean_growth_perlevel, level)
 
 
-def get_champion_base_stats(champion_stats, level):
+def get_champion_base_stats(champion_stats, level) -> StatsAnnot:
```

Comment:
```suggestion
def get_champion_base_stats(champion_stats, level) -> Stats:
```

Reply:
btw what happens if i do `from stats import Stats` if I have a `Stats` annot and a `Stats` class ?